### ISSUE TITLE
x, y, z coordinates in Vector - optionally drop the z on deserialization

### ISSUE THREAD

**Issue by moradology** [geotrellis/geotrellis#1272]
It isn't uncommon for coordinates in GeoJson to be specified with a Z value - GeoTrellis can't/won't handle the third dimension, but it would be useful to have an option which specifies automatically dropping the third dimension on JSON deserialization.

**Comment by lossyrob** [comment]
Thinking through this API

```scala
string.parseGeoJson[MultiPolygon](options = GeoJson.Options(ignoreExtraDimensions = true))
```

**Comment by lossyrob** [comment]
Fix to this would fix  #1676
